p2-maven-plugin/pom.xml

```diff
@@ -207,6 +207,11 @@
 			<groupId>commons-net</groupId>
 			<artifactId>commons-net</artifactId>
 		</dependency>
+		<dependency>
```

Comment:
p2 plugin should not depend on tycho-core

Reply:
Done

Reply:
Github still show a change here...